src/OpenTelemetry.Instrumentation.AWS/CHANGELOG.md

```diff
@@ -4,8 +4,10 @@
 
 * BREAKING (renaming): renamed `OpenTelemetry.Contrib.Instrumentation.AWS` to `OpenTelemetry.Instrumentation.AWS`
   ([#1275](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1275))
-* Raised the minimum .NET version to `net462`
+* Raised the minimum .NET Framework version to `net462`
   ([#1095](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1095))
+* Raised the minimum .NET version to `net6.0`
```

Comment:
This is not raising the .NET version of the src project.

Reply:
What is your preference, fully remove this entry and find better wording to inform that we are testing only with .net6+?

Reply:
I don't think this needs a CHANGELOG entry but if you really want to include then yes be clear about it. The current entry makes it sound like the src project is targeting `net6.0`.

Reply:
Removed 64592c3d5a2ccfe613785920e74752f1429ec418